### DISCUSSION TITLE
AUDIENCE MEMBER - trying to find information about a production seen long time ago 🐢

### DISCUSSION THREAD

**Discussion by birk** [pair-cg/general-contributions#34] [User Stories]
# V2.1 🐢

# 1. Need:

**Find production credits and possibly a video recording** of a dance production that featured a turtle slowly moving from one end of the stage to the other while the dancers were dancing. The production was a guest performance sometime between 2000-2005, in a former armory building that had been transformed into a cultural venue in the 1980s. Since 2005, the management of the venue has changed (including both the artistic director and general manager). The venue currently puts on lots of types of events such as participatory discussions and community cooking sessions. The dance company was not affiliated with the venue, but was a regular visitor with about one new production every year. 

# 2. Action:
* Lookup a copy of the printed program of the dance production in my personal belongs.
* Search using an internet based search engine for shows that took place at the venue in the given time period whose subject matter and description matches the one in my recollection e.g. "a turtle slowly moving from one end of the stage to the other while the dancers were dancing."
* Contact a local expert in dance who may recall the production (This would have been an option, if I just knew an expert I could ask.)

# 3. Desired Outcome:

### Information: 
- Found the full credits of the show 
- _Found where a particular dancer is performing now [this outcome could be broken into a whole new User Story!]_
### Artifact:
- Found a video recording of the production

# 4. Perspective:
Consumer of information - a devoted local audience member

# 5. Stakeholder Group:
* AUDIENCE MEMBER
* ARTISTIC ORGANIZATION
* ARTISTIC CREATOR
* ARCHIVIST
* TECHNOLOGY COMPANIES

# 6. Additional info:

---

# V1 (Original Contribution)

# 1. Need:
As an enthusiastic visitor of dance performances at my favorite local venue I want to find information about productions I saw there a long time ago. Unfortunately, with the recent change of management also the venue's website was relaunched and older productions are no longer listed. I am especially interested in that one production that featured a turtle slowly moving from one end of the stage to the other while the dancers were, well, dancing.

# 2. Action:
* Trying to find the printed program of that production with the turtle - without success.
* As I do not remember the exact title, I am googling all kinds of things I still remember (like what was the piece about and the turtle, of course).

# 3. Desired Outcome:
The audience member not only finds the full credits and a video recording of that production in a sustainable AKA archival location but also finds out in which company her favorite dancer is performing now.

# 4. Perspective:
Consumer of information - a devoted local audience member

# 5. Stakeholder Group:
* AUDIENCE MEMBER
* ARTISTIC ORGANIZATION
* ARTISTIC CREATOR
* ARCHIVIST

# 6. Additional info:

**Comment by samipierre** [comment]
Hi @birk 

Thank you so much for your user story. I have a couple of follow ups for you. 

1. What venue are you referring to?
2. Could you give an approximate time interval of when this show was (i.e. June 2016 or 2014-2017)?
3.  What management changed (artistic director, general manager, etc.)?
4.  Do you remember if the company that performed the show was a visiting company or hosted shows at the venue often?
5.  Do you have any more information about what types of productions this venue put on?

**Reply by birk** [reply]
OK, if you want me to further fabulate my little 🐢 story, I can.

*1. What venue are you referring to?*
It is a guest performance venue in a former armory building, which has been renovated as a cultural venue in the 1980s.

*2. Could you give an approximate time interval of when this show was (i.e. June 2016 or 2014-2017)?*
I am old, so early 2000s.

*3.  What management changed (artistic director, general manager, etc.)?*
Both! And I don't like the new artistic director and the new general manager. The program is just not as it used to be!

*4.  Do you remember if the company that performed the show was a visiting company or hosted shows at the venue often?*
I would describe them as regular visitors, so neither ensemble nor visting just once.

*5.  Do you have any more information about what types of productions this venue put on?*
They used to have really goods ones. Now that also do a lot of alternatives formats such as participatory discussions, common cooking sessions and so on.

**Reply by mariel-marshall** [reply]
Hey @birk love this example. 
When you say "regular visitor" do you mean the dance company performed at least once a year at the venue? Or they performed several times within the period of 2000-2005?

**Reply by birk** [reply]
@mariel-marshall  I'd say they came about once a year with a new production.

**Comment by KlausIllmayer** [comment]
What about an action like "search in a performance database for actor 'turtle'" (where turtle is a real animal and not a role)
